tests/main/lxd/task.yaml

```diff
@@ -89,7 +89,7 @@ execute: |
     # There isn't an official image for noble yet, let's use the community one
     REMOTE=ubuntu
     # There isn't an official image for noble yet, let's use the daily one
-    if os.query is-ubuntu 24.04; then
+    if os.query is-ubuntu 24.10; then
```

Comment:
this clearly doesn't scale, but I'm not sure how to identify a particular release as development one